config/registry/default_schema.yaml

```diff
@@ -0,0 +1,693 @@
+        A camera that defines a set of physical filters and sensors, and defines a
+        numbering system for its visits.  The SQL representation is a definition
+        table containing allowable values for e.g. Dataset.camera.
+      columns:
+      - !Column &camera
+        name: camera
+        type: string
+        primary_key: true
+  
+    - !DataUnit
+      name: AbstractFilter
+      doc: >
+        A filter that is not associated with a particular camera.  An abstract
+        filter can be used to relate similar physical filters, and is typically the
+        filter associated with coadds.  The SQL representation is a definition table
+        containing allowable values for e.g. Dataset.abstract_filter.
+      columns:
+      - !Column &abstract_filter
+        name: abstract_filter
+        type: string
+        primary_key: true
+        doc: >
+          Name of the filter, typically a single letter (e.g. "r" or "i").
+  
+    - !DataUnit
+      name: PhysicalFilter
+      dependencies:
+      - Camera
+      - AbstractFilter
+      doc: >
+        A filter associated with a particular camera.  Physical filters are used to
+        identify datasets that can only be associated with a single observation.
+        The SQL representation is is a table describing the relationships between
+        physical filters, cameras, and abstract filters.
+      columns:
+      - !Column
+        name: camera
+        type: string
+        primary_key: true
+      - !Column &physical_filter
+        name: physical_filter
+        type: string
+        primary_key: true
+      - !Column
+        name: abstract_filter
+        type: string
+      constraints:
+      - !ForeignKeyConstraint
+        src: camera
+        tgt: Camera.camera
+      - !ForeignKeyConstraint
+        src: abstract_filter
+        tgt: AbstractFilter.abstract_filter
+  
+    - !DataUnit
+      name: Sensor
+      dependencies:
+      - Camera
+      doc: >
+        A sensor associated with a particular camera (not an observation of that
+        sensor; that requires specifying an exposure or visit as well).  The SQL
+        representation is a table with metadata and grouping information, allowing
+        sensors to be identified by something other than their numbers in user
+        expressions of what to process.
+      columns:
+      - !Column
+        name: camera
+        type: string
+        primary_key: true
+        nullable: false
+      - !Column &sensor
+        name: sensor
+        type: int
+        primary_key: true
+        nullable: false
+      - !Column
+        name: name
+        type: string
+        primary_key: true
+      - !Column
+        name: group
+        type: string
+      - !Column
+        name: purpose
+        type: string
+      constraints:
+      - !ForeignKeyConstraint
+        src: camera
+        tgt: Camera.camera
+  
+    - !DataUnit
+      name: Visit
+      dependencies:
+      - Camera
+      - PhysicalFilter
```

Comment:
I think we need some way of distinguishing between these relationships in this structure:
 - `camera` is part of `Visit`'s primary key, and needs to be provided along with the `visit` number field to uniquely identify a `Visit`.
 - `physical_filter` is not a part of `Visit`'s primary key, and is not necessary to uniquely identify a `Visit`.